samples/net/sockets/http_get/src/http_get.c

```diff
@@ -25,8 +25,14 @@
 
 #if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
 #include <net/tls_credentials.h>
+#if defined(CONFIG_TLS_CREDENTIAL_FILENAMES)
+static const unsigned char ca_certificate[] = "globalsign_r2.der";
+#else
 #include "ca_certificate.h"
 #endif
+#define CA_CERTIFICATE_TAG 1
+static const unsigned int ca_cert_len = sizeof(ca_certificate);
```

Comment:
Do we need to introduce this variable at all? In both cases we store there `sizeof(ca_certificate)`.

Reply:
Yup, not needed.  Can restore to what it was.